includes/base/controls-stack.php

```diff
@@ -707,6 +716,28 @@ public function get_stack() {
 		);
 	}
 
+	/**
+	 * Convert Scheme to Default Global
+	 *
+	 * If a control has a scheme property, convert it to a default Global.
+	 *
+	 * @param $args - control/group control args
+	 * @return array - control/group control args
+	 * @since 3.0.0
+	 * @access public
+	 *
+	 */
+	public function convert_scheme_to_default_global( $args ) {
+		if ( isset( $args['scheme']['type'] ) && isset( $args['scheme']['value'] ) ) {
```

Comment:
Please prepare a deprecation notice with `_deprecated_argument`.

Reply:
Not sure I understand, this is a new method

Reply:
A developer that uses the `scheme` argument (on `add_control`) should see a notice that it's a deprecated argument.
Our standard is to prepare the notice in a comment, and after 4 versions we show it.

Reply:
Do I put this in every location where $args['scheme'] is used?

Reply:
Only here, because every use of `scheme` run's this method. I suggest to pass additional arguments, the element and control names in order to add theme in the notice content.